KinConnections/templates/connectors.html

```diff
@@ -10,116 +10,92 @@ <h2>Connectors List and Search</h2>
-                <li data-filter=".filter-Urduhindi">Urdu/Hindi</li>
-              </ul>
+            <div class="row" data-aos="fade-up" data-aos-delay="100">
+              <div class="col-lg-12">
+                <h4>Filter by Language</h4>
+                <ul id="portfolio-flters" data-filter-group="language">
+                  <li data-filter="">All Languages</li>
+                  <li data-filter=".filter-Arabic">Arabic</li>
+                  <li data-filter=".filter-Dari">Dari</li>
+                  <li data-filter=".filter-English">English</li>
+                  <li data-filter=".filter-Farsi">Farsi</li>
+                  <li data-filter=".filter-French">French</li>
+                  <li data-filter=".filter-Gujarati">Gujarati</li>
+                  <li data-filter=".filter-Kutchi">Kutchi</li>
+                  <li data-filter=".filter-Portuguese">Portuguese</li>
+                  <li data-filter=".filter-Russian">Russian</li>
+                  <li data-filter=".filter-Spanish">Spanish</li>
+                  <li data-filter=".filter-Urduhindi">Urdu/Hindi</li>
+                </ul>
+              </div>
             </div>
-          </div>
 
-          
-  
-          <div class="row portfolio-container" data-aos="fade-up" data-aos-delay="200">
-            {% for connector in connectors %}
-            <!-- start repeat -->
-            <div class="col-lg-4 col-md-6 portfolio-item {% for professional_filter in connector['professional_category_filters'] %} {{professional_filter}} {% endfor %} {% for language_filter in connector['language_category_filters'] %} {{language_filter}} {% endfor %}">
-              <div class="portfolio-wrap">
-                <img src="{{connector['images']}}" class="img-fluid" alt="">
-                <div class="portfolio-info">
-                  <p style="font-size: 22px; font-weight: 700; font-family: 'Montserrat', sans-serif;">
-                    <a href="{{connector['images']}}" data-gallery="portfolioGallery" title="{{connector.get('first_name', 'Error')}} {{connector.get('last_name', 'Error')}} - {{connector.get('title', 'Error')}}" class="portfolio-lightbox">
-                      {{connector.get('first_name', "Error")}} {{connector.get('last_name', "Error")}}
-                    </a>
-                  </h4>
-                  <!-- <p>{{connector.get('title', "Error")}}</p> -->
-                  <div>
-                    <a href="{{connector['images']}}" data-gallery="portfolioGallery" title="{{connector.get('first_name', 'Error')}} {{connector.get('last_name', 'Error')}} - {{connector.get('title', 'Error')}}" class="link-preview portfolio-lightbox"><i class="bi bi-plus"></i></a>
-                    <a href="/connectors/{{connector['id']}}" class="link-details" title="More Details"><i class="bi bi-link"></i></a>
+            <div class="row portfolio-container" data-aos="fade-up" data-aos-delay="200">
+              <ul id="connectorsList">
```

Comment:
Seeing an issue somewhere here that also needs to be looked at more. The flex-box, which is what I think this is, gets messed up once the search takes place. It jumbles up the connectors and really butchers the UI. This might be a simple fix. Haven't really looked too much into this.

Reply:
Fixed this with recent commits. Had to make lots of changes to the HTML code. Will do some thorough testing on this.